webfront/views/custom.py

```diff
@@ -353,10 +395,47 @@ def update_queryset_from_search(self, searcher, general_handler):
         self.before_key = before_key
         self.elastic_result = elastic_result if should_keep_elastic_order else None
 
+    def update_queryset_from_taxonomy_per_entry(self, general_handler):
+        entry_acc = general_handler.queryset_manager.filters["entry"]["accession"]
+        self.queryset = TaxonomyPerEntry.objects.filter(
+            entry_acc=entry_acc.upper()
+        ).order_by("num_proteins")
```

Comment:
Is it relevant to order by `num_proteins` here, since we're going to re-order by `-num_proteins` in the taxonomy/proteome views?

Reply:
mhh... it actually needs to have an `order_by` here because, in the custom paginator, we default to order by accession, and this table doesn't have `accession` so it fails if left empty. 

But it actually should be `"-num_proteins"` to be consistent.

TBH, I was surprised that I [needed to pass](https://github.com/ProteinsWebTeam/interpro7-api/blob/1934d10173dcf5a2d1b5c7122077a39f0a1ee42d/webfront/views/custom.py#L166) `ordering` to the paginator to work even if the `queryset` already has an `order_by`. I think it uses the given queryset without actually doing the query for all the precalculations, e.g. number of pages, etc. and then it actually executes the query using the calculated parameters(e.g. page, cursor, etc.), and that's when it uses the `ordering`.